oneandone/resource_oneandone_baremetal_server.go

```diff
@@ -0,0 +1,370 @@
+package oneandone
+
+import (
+	"fmt"
+	"log"
+	"strings"
+
+	"github.com/1and1/oneandone-cloudserver-sdk-go"
+	"github.com/hashicorp/terraform/helper/schema"
+)
+
+func resourceOneandOneBaremetalServer() *schema.Resource {
```

Comment:
The server will get deleted in about a minute or two, we can poll it and check its status. However the server requires 24 hours to be available again and get back the pool. The idea is to avoid stressing 1&1's capacity of baremetal servers and  to not increase test time significantly

Reply:
That seems a fair tradeoff 👍